cmd/whereabouts_test.go

```diff
@@ -234,6 +234,20 @@ var _ = Describe("Whereabouts operations", func() {
 		)
 	})
 
+	It("allocates and releases an IPv6 range that ends with zeroes with a Kubernetes backend", func() {
+
+		ipVersion := "6"
+		ipRange := "2001:1b74:480:603d:0304:0403:000:0000-2001:1b74:480:603d:0304:0403:0000:0004/64"
```

Comment:
Let's use a documentation range, so make the range start with ` 2001:db8`

Also, add an additional test that doesn't just ends with zeros, but another one that ends with `::` literally so, for example.... `2001:db8:b33f::/48` or something

Reply:
see also: https://www.rfc-editor.org/rfc/rfc3849.txt